src/Features/VisualBasic/Portable/Wrapping/InitializerExpression/AbstractVisualBasicInitializerExpressionWrapper.vb

```diff
@@ -0,0 +1,21 @@
+﻿' Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+Imports Microsoft.CodeAnalysis
+Imports Microsoft.CodeAnalysis.VisualBasic.Indentation
+Imports Microsoft.CodeAnalysis.Wrapping.InitializerExpression
+
+Friend MustInherit Class AbstractVisualBasicInitializerExpressionWrapper(
+    Of TListSyntax As SyntaxNode, TListItemSyntax As SyntaxNode)
```

Comment:
Do we ever split the type parameters like this on a second line anywhere else? It definitely made me do a double take. I'm happy for this to stay on the prior line.

Reply:
we do occasionally. usually when the length becomes more than 120 chars.  In this case, we can likely just keep it on one line.  However, this is not out of hte norm for the codebase.